### ISSUE TITLE
Dark mode menu/tray icon

### ISSUE THREAD

**Issue by Phidica** [samuelthomas2774/nxapi#49]
The current tray icon is completely black, meaning that against trays that are also black (eg, Gnome on Linux) the icon is completely invisible (at least until interacted with):

![Screenshot from 2023-01-15 22-06-11](https://user-images.githubusercontent.com/13155128/212538327-dfdfc3b5-e603-467f-8110-c16fe65f223d.png) ![Screenshot from 2023-01-15 22-06-07](https://user-images.githubusercontent.com/13155128/212538326-82b59cec-461f-4433-b0a8-154b8749a714.png)

Can there be a toggle in the settings for selecting between a light and dark mode tray icon? For a dark mode tray icon, I think a simple inversion of the existing icon (so that it is grey/white) would be fine.

Thanks for the great program!

**Comment by Roadhog360** [comment]
Perhaps the simplest solution could be adding a subtle white outline to the icon so one icon can be used against solid white or solid black backgrounds.

**Comment by hilot06** [comment]
I have same problem, and I use this icon instead
![menu-icon.png](https://github.com/samuelthomas2774/nxapi/assets/118000723/d6d2b8c6-1e99-4d5a-95c0-f0d931123c8d)
![image](https://github.com/samuelthomas2774/nxapi/assets/118000723/54c9c869-8d62-4b54-a3ac-c394e2c13cac)

And I tried Roadhog360's idea
![menu-icon.png](https://github.com/samuelthomas2774/nxapi/assets/118000723/5aaff0e1-c3c6-4dd0-9f42-7499af7ce12a)
![image](https://github.com/samuelthomas2774/nxapi/assets/118000723/413e4947-c4fb-40c3-9be2-654fdcfae1a9)

**Comment by cobalt2727** [comment]
> I have same problem, and I use this icon instead ![menu-icon.png](https://user-images.githubusercontent.com/118000723/246349018-d6d2b8c6-1e99-4d5a-95c0-f0d931123c8d.png)

I'd say go with this one then just fill in the space inside the actual icon so that the only transparent part of the image is outside the borders

**Comment by hilot06** [comment]
like this?
![menu-icon_3](https://github.com/samuelthomas2774/nxapi/assets/118000723/875d24db-0a86-470c-a4a8-6ccfa7e66a17)
(Note: It may look bad in light mode)

**Comment by hilot06** [comment]
looks good
![image](https://github.com/samuelthomas2774/nxapi/assets/118000723/9319e8f0-998e-4838-bd5f-22387cecda29)

**Comment by cobalt2727** [comment]
Nah, I'm saying fill in the black on the left one, fill in the space between them, and also fill in the control stick/buttons on the right

**Comment by CarJem** [comment]
Why is this not a thing yet? At least give us a manual option like so many other linux apps do!
